en/rocky/8/admin_guide/05-vi.md

```diff
@@ -0,0 +1,479 @@
+# VI Text Editor
+
+In this chapter you will learn how to work with the VIsual editor.
+
+****
+**Objectives** : In this chapter, futurs Linux administors will learn how to:
+
+:heavy_check_mark: Use the main commands of the VI editor; \
+:heavy_check_mark: Modify a text with the VI editor. \
+
+:checkered_flag: **user commands**, **linux**
+
+**Knowledge**: :star: \
+**Complexity**: :star: :star:\
+**Reading time**: 20 minutes
+****
+
+*Visual* (**VI**) is a very popular text editor under Linux, despite its limited ergonomics. It is indeed an editor entirely in text mode: each action is done with a key on the keyboard or dedicated commands.
+
+Very powerful, it is above all very practical since it is on the whole minimal for basic applications. It is therefore accessible in case of system failure. Its *universality* (it is present on all Linux distributions and under Unix) makes it a *crucial* tool for the administrator.
```

Comment:
?

Reply:
What did you mean ?

Reply:
difficult to understand (for me)

Reply:
Let's wait for @sspencerwire what he suggests